R/as.SpatRaster.R

```diff
@@ -0,0 +1,57 @@
+#' as.SpatRaster
+#'
+#' Convert magclass object to a  object
+#'
+#'
+#' @param x MAgPIE object
+#' @param res spatial data resolution. If not provided it will be guessed.
+#' @return A  object
+#' @author Jan Philipp Dietrich
+#' @seealso \code{\link{getCoords}}
+#' @examples
+#'
+#' if (requireNamespace("raster", quietly = TRUE)) {
+#'    r <- raster::brick(ncols = 360, nrows = 180, nl = 4)
+#'    r[85:89, 176:179] <- (1:20 %*% t(1:4))
+#'    r[15:19, 76:79] <-   (10 + 1:20 %*% t(1:4))
+#'
+#'    names(r) <- c("y2000..bla", "y2001..bla", "y2000..blub", "y2001..blub")
+#'    m <- as.magpie(r)
+#'    r2 <- as.SpatRaster(m)
+#' }
+#' @export
+
+
+as.SpatRaster <- function(x, res = NULL) { # nolint
```

Comment:
Assuming you only need to disable the camelCase linter warning:
```suggestion
as.SpatRaster <- function(x, res = NULL) { # nolint: object_name_linter
```